client/src/components/Counter.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ReactComponent as HeartIcon } from "../assets/icons/heart-icon.svg";
+import styled from "@emotion/styled";
+
+export default function Counter(props) {
+  return (
+    <CounterStyle>
+      <Heart>
+        <HeartIcon />
+      </Heart>
+      <output>{props.number}</output>
+    </CounterStyle>
+  );
+}
+
+Counter.propTypes = {
+  number: PropTypes.func,
```

Comment:
number doesn't sound like a func, more like a number. If so, you should change the prop validation to 
```suggestion
  number: PropTypes.num,
```
If not, maybe you could rename number to something like `onSetNumber`

Reply:
Thanks, good shout on this!